packages/components/src/autocomplete/style.native.scss

```diff
@@ -0,0 +1,73 @@
+.wrapper {
```

Comment:
I think we should start following the new BEM stylesheet pattern in the new document.

Reply:
This sounds interesting. Is there an example of BEM usage on the RN side of things? I know there is already some "magic" happening between kebab-case and camelCase things via our scss transpilation (since JS can't have variables / property names with `-`), but curious about the _new_ pattern you mention.

Reply:
Thanks @enejb for the suggestion, we started this feature before deciding about using the BEM pattern but we'll look into it.

Reply:
Updated it here https://github.com/WordPress/gutenberg/pull/29772/commits/47bd27e56ffc0a1d5c2ab5da36735603457e0201